docs/content/en/docs/topics/provision-a-cluster/_index.md

```diff
@@ -53,42 +53,35 @@ helm get manifest demo | grep size
 .
 .
 .
-  size: 1
-      initial_heap_size: "800M"
-      max_heap_size: "800M"
-    :[{\"expr\":\"sum(mcac_table_memtable_off_heap_size{cluster=~\\\"$cluster\\\"\
-    :\"A\"},{\"expr\":\"sum(mcac_table_memtable_on_heap_size{cluster=~\\\"$cluster\\\
-    ,\"description\":\"Total sizes of the data on distinct nodes\",\"fill\":0,\"gridPos\"\
-    datasource\":\"$PROMETHEUS_DS\",\"description\":\"Maximum JVM Heap Memory size\
-    \ (worst node) and minimum available heap size\",\"fill\":1,\"gridPos\":{},\"\
+    size: 1
+      initial_heap_size: 1G
+      max_heap_size: 1G
+      heap_size_young_generation: 1G
 ```
 
-Notice the value of `size: 1` from cassdc.yaml. This is the Cassandra DataCenter definition. 
+The value of `size: 1` is from cassdc.yaml, which is the Cassandra DataCenter definition. 
```

Comment:
`Cassandra Datacenter` should be written as one word, `CassandraDatacenter`.

Reply:
Done, thanks.